src/components/PokemonLogo/PokemonLogo.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from "react";
+
+export type PokemonLogoProps = React.SVGProps<SVGSVGElement> & {
+  size?: number;
+  imgSrc?: string;
+};
+
+export const PokemonLogo: React.FC<PokemonLogoProps> = ({
+  size = 48,
+  imgSrc,
+  ...props
+}) =>
```

Comment:
React.FC is depreacted

```suggestion
export const PokemonLogo = ({
  size = 48,
  imgSrc,
  ...props
}: PokemonLogoProps) =>
```

Reply:
Changed.